### PR TITLE
Missed search by address for keystore

### PR DESCRIPTION
**Pull request by r8d8**
Added funcitonality for search keystore files by address in given path

### CODE REVIEW

File: tests/keystore_test.rs [hunk 1]
```diff
@@ -180,6 +180,38 @@ fn should_create_keyfile() {
...
+fn should_search_by_address() {
+    let addr = "0x0047201aed0b69875b24b614dda0270bcd9f11cc"
+        .parse::<emerald::Address>()
+        .unwrap();
+
+    let res = search_by_address(&keystore_path(), &addr);
+    assert!(res.is_some());
+
+    let kf = res.unwrap();
+    assert_eq!(kf.cipher_text,
```
Comment (dulanov) [inline tests/keystore_test.rs]: Checking uuid will be anough?

Comment (r8d8) [inline tests/keystore_test.rs]: You mean, it is necessary to add assert for each field ?

Comment (dulanov) [inline tests/keystore_test.rs]: No i mean it's anough just to proof `uuid` from `KeyFile` to be sure that all right.

Comment (r8d8) [inline tests/keystore_test.rs]: Done

**Pull request merged**